ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tunnel-interfaces.j2

```diff
@@ -0,0 +1,88 @@
+{% if tunnel_interfaces is defined and tunnel_interfaces is not none %}
```

Comment:
```suggestion
{% if tunnel_interfaces is arista.avd.defined %}
```

Reply:
This comes from `documentation/vlan-interfaces.j2` should I clean it as well?

Reply:
Addressed in 0db9c4a